<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IngresoDialog</class>
 <widget class="QDialog" name="IngresoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingreso sorteo</string>
  </property>
  <widget class="QLineEdit" name="lN13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN12">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NÃºmero de Sorteo</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN09">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN14">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN01">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN06">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN08">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN05">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN11">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN03">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN07">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN02">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lsorteo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN04">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Sorteo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lN10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>lsorteo</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lN01</tabstop>
  <tabstop>lN02</tabstop>
  <tabstop>lN03</tabstop>
  <tabstop>lN04</tabstop>
  <tabstop>lN05</tabstop>
  <tabstop>lN06</tabstop>
  <tabstop>lN07</tabstop>
  <tabstop>lN08</tabstop>
  <tabstop>lN09</tabstop>
  <tabstop>lN10</tabstop>
  <tabstop>lN11</tabstop>
  <tabstop>lN12</tabstop>
  <tabstop>lN13</tabstop>
  <tabstop>lN14</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
